platform/src/main/java/org/stellar/anchor/platform/utils/TransactionHelper.java

```diff
@@ -66,10 +66,18 @@ static GetTransactionResponse toGetTransactionResponse(
       refunds = toRefunds(txn.getRefunds(), txn.getAmountInAsset());
     }
 
+    String sender = DEPOSIT.kind.equals(txn.getKind()) ? txn.getToAccount() : txn.getFromAccount();
```

Comment:
The sending and receiving customers are identified by the account authenticated in SEP-10, not the destination account for the payment. Also note that customers of custodial wallets are identified by an account AND memo.

Reply:
I think there is a misunderstanding on my side. If we use customers for this purpose, where should the payment destination be set?

Reply:
Ah.. we don't have field for it. We should probably just add `destination_account` and `source_account` fields to the top-level transaction object.